network_diffusion/seeding/drivernode_selector.py

```diff
@@ -0,0 +1,82 @@
+"""A definition of the seed selector based on driver nodes."""
```

Comment:
And now the ultimate dilemma: since not a node, but an actor is a base agent in our library, should we change the naming of `DriverNodes` to `DriverActors` in the entire codebase? What do you think @uniooo?

Reply:
Yes, I had the same thought. I believe a node pertains to a single layer, while actors are for multi-layer networks. So, let's call them DriverActors. What do you think, @anty-filidor?